libs/stache/src/lib/modules/affix/affix-top.directive.ts

```diff
@@ -53,17 +56,21 @@ export class StacheAffixTopDirective implements AfterViewInit {
     }
   }
 
+  public ngAfterContentInit(): void {
+    this.onWindowScroll();
```

Comment:
I came across a weird jumping issue where things previously might not be considered affixed on load when they should have been. This caused an odd jitter on the first scroll where the element would scroll and then jump back. This addresses that.

Reply:
The Wrapper playground page showed the jitter. I added a comment.